### DISCUSSION TITLE
"Browse..." Icon still showing and added images dont show in the UI

### DISCUSSION THREAD

**Discussion by blufox** [spatie/laravel-medialibrary#2771] [Q&A]
Im struggling to get the Media Library Pro working properly. On the page the Media Library Pro UI seems to be having an issue. Firstly the "Browse.." icon that appears from normal uploads is still there after the installation process. The next issue is when I add an image the thumbnail doesnt appear. After trying to submit what is there i get an error "The images must be an image". I should mention Im also new to laravel so a may be asking a dumb question here.

Link to the image of the UI here:
[https://paste.pics/dff85b7b793b9dcccb46a9e701b7b773](url)

This is my blade script:
`<x-media-library-collection name="images" rules="mimes:jpg,jpeg,png" :model="$album" collection="gallery"/>`

This is my model script:
```
use Spatie\MediaLibrary\HasMedia;
use Spatie\MediaLibrary\InteractsWithMedia;
use Spatie\MediaLibrary\MediaCollections\Models\Media;
class Album extends Model implements HasMedia
{
    use HasFactory, InteractsWithMedia;


    //fillable
    protected $fillable = ['album_title', 'album_prefix', 'album_slug'];

    public function registerMediaCollections(): void
    {
        $this->addMediaCollection('gallery');         

    }
    
    public function gallery()
    {
        return $this->hasMany(Gallery::class);
    }
}
```

And my controller:
```
public function store(Request $request)
    {
        $this->validate($request, [
            'images' => 'required|image|mimes:jpeg,png,jpg,gif,svg|max:2048',
        ]);

        $album = Album::findOrFail($request->input('id'));        

        $album
        ->syncFromMediaLibraryRequest($request->file('images'))
        ->toMediaCollection('gallery');

        return redirect()->route('gallery.index', $album->id)->with('success', 'Image Uploaded Successfully');
    }
```

Any help would be much appreciated.

**Comment by freekmurze** [comment]
Try to remove that `image` validation rule. That request key won't hold an image, but a reference to a temporary uploaded.

To see some example on how to use media library pro, take a look at our demo app: https://github.com/spatie/laravel-medialibrary-pro-app
